### DISCUSSION TITLE
Website zshell.dev is DOWN

### DISCUSSION THREAD

**Discussion by rot26** [z-shell/zi#193] [General]
I went to the website looking for documentation.  Looks like it's down.

All I can see is `Error thrown object is not defined`.  I checked in a different browser and even via `curl`

https://wiki.zshell.dev/

```
$ curl https://wiki.zshell.dev/
```

```
Error thrown object is not defined
```

**Comment by rot26** [comment]
Ignore. It's back up now.  Maybe some blue-green deployment, or a staging environment would help so prod doesn't go down :)

**Reply by ss-o** [reply]
Apologies, the unexpected issue, shouldn't happen again :ballot_box_with_check: :octocat:
